FAQ.md

```diff
@@ -0,0 +1,49 @@
+SignalR Frequently Asked Questions
+=======
+
+This document will condense some of the more popular questions and themes that have come up in the issues we've received on SignalR. Please check the FAQ before you submit a new issue, as the issue may have already been addressed below. 
+
+---
+
+**What is SignalR?**
+
+SignalR is a series of libraries for both the server and client that make it incredibly simple to add real-time web functionality to your apps. You can learn more about SignalR in the [SignalR documentation](https://docs.microsoft.com/en-us/aspnet/core/signalr/introduction). 
```

Comment:
Isn't it more than "web" now? E.g. Java client on Android, and serverless -- no web.

Reply:
couldn't agree more. update OTW will have this removed.